src/main.rs

```diff
@@ -1,3 +1,114 @@
+extern crate gio;
+extern crate gtk;
+
+use gio::prelude::*;
+use gio::MenuExt;
```

Comment:
Why is `MenuExt` no in the `gio` prelude?

Reply:
This is a pretty good question!

Reply:
We should fix that :)

Reply:
Indeed but that won't get in before rustfest I assume.

Reply:
Not a new release, yes. Unless you do a bugfix release for that (I do bugfix releases for such things and lots of other stuff in gstreamer-rs regularly, see https://github.com/sdroege/gstreamer-rs/blob/master/gstreamer/CHANGELOG.md)

Reply:
Let's say I'll think about. :-.